db/migrations/20200120-drop-users.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE jobs DROP COLUMN user_id;
+DROP TABLE users;
```

Comment:
This migration is done outside of Gorm's auto-migrate.  Should the upcoming release ticket be updated to indicate that this script should be run in `dev`, `test`, `opensbx`, and `prod`?

Reply:
Yes, immediately after deployment and before running a smoke test.  Ticket is updated.